src/lib/actions/sendEvent/createSendEvent.js

```diff
@@ -10,12 +10,18 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-module.exports = ({ instanceManager, turbine }) => settings => {
+module.exports = ({
+  instanceManager,
+  turbine,
+  decisionsCallbackStorage
+}) => settings => {
   const { instanceName, ...otherSettings } = settings;
   const instance = instanceManager.getInstance(instanceName);
 
   if (instance) {
-    instance("sendEvent", otherSettings);
+    instance("sendEvent", otherSettings).then(decisions => {
+      decisionsCallbackStorage.triggerEvent(decisions);
```

Comment:
In some cases decisions are not returned from sendEvent right?  Should we add a check here and only trigger the event if there are decisions?

Reply:
yes, I think it makes sense to check here rather than ask the customer to check for decisions in his custom code (of decisions received event).